### PR TITLE
Get rid of react tools

### LINKED ISSUE #227: React-tools is deprecated
I notice a reference in package.json to react-tools, which has [been deprecated](https://facebook.github.io/react/blog/2015/06/12/deprecating-jstransform-and-react-tools.html). Should this be removed and replaced with Babel?

### PR DESCRIPTION
**Pull request by apechimp**
React tools is deprecated. This replaces react tools for babel and updates the
version of react used in the tests to 0.15.

This resolves #227.

### CODE REVIEW

**Comment by apechimp** [comment]
The build is failing, but only on node 0.10 and 0.12. The build is failing on master for these runtimes as well.

**Comment by colinsullivan** [comment]
+1 this worked for me.  Otherwise using new React doesn't work.

**Comment by AndreLion** [comment]
Hi @apechimp and any moderators, any update or progress on this PR?
It will be appreciated if this PR could be merged and make new `react v15` running 😃

**Comment by apechimp** [comment]
No updates from me. I created it and it's been quiet from the maintainer since.

File: lib/consolidate.js [hunk 1]
```diff
@@ -1179,14 +1179,14 @@ function reactRenderer(type){
         if (!cache(options)){
           // Parsing
           Code = (type === 'path') ? require(resolve(str)) : requireReactString(str);
```
Comment (AndreLion) [inline lib/consolidate.js]: `require(resolve(str))` throws out error when using es6 `Class` module
>Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object

[some similar issue](https://github.com/jedireza/hapi-react-views/issues/44)

change to `require(resolve(str)).default` could fix this error but may be bringing in new error for es5

**Comment by niftylettuce** [comment]
@apechimp can you check the latest from v.0.14.2 and remerge?

**Comment by apechimp** [comment]
@niftylettuce, I merged master into this branch.

**Comment by niftylettuce** [comment]
Thanks, published `v.0.14.3`

**Comment by niftylettuce** [comment]
Actually, I had to unpublish it - @apechimp can you fix the broken tests?

**Comment by apechimp** [comment]
Oh no! Which broken tests? The CI build passes.

**Comment by apechimp** [comment]
Ahh, I see, it's [this build](https://travis-ci.org/tj/consolidate.js/builds/175143464) that failed. It looks like that is some kind of git issue within travis.

**Pull request merged**